### ISSUE TITLE
[millipede] broken example (Trac #1253)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1247]
the example resources/examples/millipede-angular.py fails with this traceback:

Traceback (most recent call last):
  File "millipede/resources/examples/millipede-angular.py", line 37, in <module>
    StepAzimuth=5*I3Units.deg,
  File "/home/hdembinski/Code/icecube/icerec/debug.trunk/lib/I3Tray.py", line 204, in AddSegment
    return _segment(self, _name, **kwargs)
  File "/home/hdembinski/Code/icecube/icerec/debug.trunk/lib/icecube/millipede/__init__.py", line 235, in MillipedeFit
    tray.AddService('MillipedeLikelihoodFactory', likelihood, **millipede_config)
  File "/home/hdembinski/Code/icecube/icerec/debug.trunk/lib/I3Tray.py", line 167, in AddService
    super(I3Tray, self).SetParameter(_name, k, v)
RuntimeError: Attempt to set parameter ShowerSpacing that doesn't exist (in void I3ConfigurationImpl::Set(const string&, const boost::python::api::object&))

Migrated from https://code.icecube.wisc.edu/ticket/1253
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "the example resources/examples/millipede-angular.py fails with this traceback:\n\nTraceback (most recent call last):\n  File \"millipede/resources/examples/millipede-angular.py\", line 37, in <module>\n    StepAzimuth=5*I3Units.deg,\n  File \"/home/hdembinski/Code/icecube/icerec/debug.trunk/lib/I3Tray.py\", line 204, in AddSegment\n    return _segment(self, _name, **kwargs)\n  File \"/home/hdembinski/Code/icecube/icerec/debug.trunk/lib/icecube/millipede/__init__.py\", line 235, in MillipedeFit\n    tray.AddService('MillipedeLikelihoodFactory', likelihood, **millipede_config)\n  File \"/home/hdembinski/Code/icecube/icerec/debug.trunk/lib/I3Tray.py\", line 167, in AddService\n    super(I3Tray, self).SetParameter(_name, k, v)\nRuntimeError: Attempt to set parameter ShowerSpacing that doesn't exist (in void I3ConfigurationImpl::Set(const string&, const boost::python::api::object&))\n", 
    "reporter": "hdembinski", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[millipede] broken example", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-20T19:03:03", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150824T19:58:32: 
* jbraun changed resolution from "" to "fixed"
* jbraun changed status from "new" to "closed"
* jbraun commented:

> In changeset 136808:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136808"
> Rename MuMillipedeFit parameter 'CascadeSpacing' --> 'ShowerSpacing' to agree with millipede conventions.  Closes #1247
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
